### PR TITLE
31446 better labels

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile/components/ProfileMobileSubNav.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const ProfileMobileSubNav = ({ isLOA3, isInMVI, routes }) => {
 
   // on first render, set the focus to the h4
   useEffect(() => {
-    focusElement('#mobile-subnav-header');
+    focusElement('#mobile-subnav-header-button');
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/ProfileMobileSubNav.jsx:26]: could you update the comment two lines above this?

File: src/applications/personalization/profile/components/ProfileSubNav.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ const ProfileSubNav = ({ isInMVI, isLOA3, routes }) => {
   }, []);
 
   return (
-    <nav className="va-subnav" aria-label="Secondary">
+    <nav className="va-subnav" aria-labelledby="subnav-header">
```
Comment (erikphansen) [inline src/applications/personalization/profile/components/ProfileSubNav.jsx:15]: could you update the comment above the `useEffect` hook?

**Review by erikphansen — COMMENTED** [review]
looks good! I just made a couple of comments re: updating some code comments that should have been updated in a previous PR.

File: src/applications/personalization/profile/components/ProfileMobileSubNav.jsx (hunk 1 above)
Comment (tmitchellgcio) [reply src/applications/personalization/profile/components/ProfileMobileSubNav.jsx:26]: completed

File: src/applications/personalization/profile/components/ProfileSubNav.jsx (hunk 2 above)
Comment (tmitchellgcio) [reply src/applications/personalization/profile/components/ProfileSubNav.jsx:15]: completed

**Review by erikphansen — APPROVED** [review]

**Pull request merged**